src/Microsoft.ML.Transforms/Text/WordEmbeddingsTransform.cs

```diff
@@ -442,4 +498,95 @@ private Model GetVocabularyDictionary()
             }
         }
     }
+
+    public sealed class WordEmbeddingsEstimator : TrivialEstimator<WordEmbeddingsTransform>
```

Comment:
>WordEmbeddingsEstimator [](start = 24, length = 23)

I'm not certain that this is a trivial estimator, since loading the model file is a fairly elaborate operation.

Reply:
I got the understanding that any transform that does not require training falls into `trivial estimator` category. That's where `WordEmbedding` transform falls into. Why do you think its not `trivial`? what other option are there for this?

---
In reply to: [218166505](https://github.com/dotnet/machinelearning/pull/928#discussion_r218166505) [](ancestors = 218166505)

Reply:
You can always push loading of data to mapper. 

---
In reply to: [218247817](https://github.com/dotnet/machinelearning/pull/928#discussion_r218247817) [](ancestors = 218247817,218166505)

Reply:
Yes, that pretty much whats happening right now in this transform.

---
In reply to: [218250223](https://github.com/dotnet/machinelearning/pull/928#discussion_r218250223) [](ancestors = 218250223,218247817,218166505)